packages/grafana-ui/src/components/Table/Table.tsx

```diff
@@ -110,8 +110,25 @@ export const Table = memo((props: Props) => {
     [data, width, columnMinWidth, footerItems, hasNestedData, isCountRowsSet]
   );
 
+  const toggleAllRowsExpandedRef = useRef<(value?: boolean) => void>();
```

Comment:
I think this needs a bit more explanation either in code comments or in the PR of why do we need the ref for the function here as it's not right away clear.